read.go

```diff
@@ -155,6 +155,10 @@ func (r *reader) readIndexData(toc *indexTOC) (*indexData, error) {
 	d.metaData = *md
 	d.repoMetaData = []Repository{*repo}
 
+	for _, md := range d.repoMetaData {
+		md.RawConfigEncoded = EncodeRawConfig(md.RawConfig)
```

Comment:
This is a no-op, since md is a copy of an element (`d.repoMetadata` is `[]Repository`)

I think not setting md.RawConfigEncoded will make all repos always visible.

Reply:
Good catch!

Reply:
Thinking about it, this should better move into readMetaData.